WindowsFirewallHelper/WindowsFirewallHelper.csproj

```diff
@@ -34,6 +33,16 @@
       <PackagePath>\</PackagePath>
     </Content>
   </ItemGroup>
+  <ItemGroup Condition="'$(TargetFramework)' == 'netstandard2'">
```

Comment:
these two conditions can be merged together.

Reply:
Assembly System.ServiceProcess.ServiceController ver 4.7.0 not compatible with net5